pandas/tests/indexing/test_floats.py

```diff
@@ -59,115 +59,117 @@ def test_scalar_error(self, index_func):
         with pytest.raises(TypeError, match=msg):
             s.iloc[3.0] = 0
 
-    def test_scalar_non_numeric(self):
-
-        # GH 4892
-        # float_indexers should raise exceptions
-        # on appropriate Index types & accessors
-
-        for index in [
+    @pytest.mark.parametrize(
```

Comment:
eventually consolidate move to top-level (as i suspect we have many of these fixtures, or will as you parametrize)